conf/ansible/inventory/group_vars/all.yaml

```diff
@@ -22,3 +22,27 @@ cron:
   http_proxy:
   https_proxy:
   no_proxy:
+
+aws_elb_account_ids:
```

Comment:
Please add a comment with a link to the source of these account IDs. So that if someone needs to add a new region in the future, it's easy to find the source of truth.